db/migrate/20170626162941_create_users.rb

```diff
@@ -1,7 +1,19 @@
 class CreateUsers < ActiveRecord::Migration[5.1]
   def change
     create_table :users do |t|
-
+	t.string :fname
```

Comment:
Code is much more readable when you take the time to type out variable names and not abbreviate things: `first_name`, `last_name`, etc. Good job switching `pnum` to `phone_number`